gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributeViewTest.java

```diff
@@ -49,67 +47,76 @@
+      Files.write(path, HAPPY, withCacheControl("potato"));
+      CloudStorageFileAttributeView lazyAttributes =
+          Files.getFileAttributeView(path, CloudStorageFileAttributeView.class);
+      assertThat(lazyAttributes.readAttributes().cacheControl().get()).isEqualTo("potato");
+    }
   }
 
   @Test
   public void testReadAttributes_notFound_throwsNoSuchFileException() throws IOException {
-    CloudStorageFileAttributeView lazyAttributes =
-        Files.getFileAttributeView(path, CloudStorageFileAttributeView.class);
-    thrown.expect(NoSuchFileException.class);
-    lazyAttributes.readAttributes();
+    try (FileSystem fs = helper.forBucket("red")) {
+      Path path = fs.getPath("/water");
+      CloudStorageFileAttributeView lazyAttributes =
+          Files.getFileAttributeView(path, CloudStorageFileAttributeView.class);
+      thrown.expect(NoSuchFileException.class);
+      lazyAttributes.readAttributes();
+    }
   }
 
   @Test
   public void testReadAttributes_pseudoDirectory() throws IOException {
-    Path dir = Paths.get(URI.create("gs://red/rum/"));
-    CloudStorageFileAttributeView lazyAttributes =
-        Files.getFileAttributeView(dir, CloudStorageFileAttributeView.class);
-    assertThat(lazyAttributes.readAttributes())
-        .isInstanceOf(CloudStoragePseudoDirectoryAttributes.class);
+    try (FileSystem fs = helper.forBucket("red")) {
+      Path dir = fs.getPath("/rum/");
+      CloudStorageFileAttributeView lazyAttributes =
+          Files.getFileAttributeView(dir, CloudStorageFileAttributeView.class);
+      assertThat(lazyAttributes.readAttributes())
+          .isInstanceOf(CloudStoragePseudoDirectoryAttributes.class);
+    }
   }
 
   @Test
   public void testName() throws IOException {
-    Files.write(path, HAPPY, withCacheControl("potato"));
-    CloudStorageFileAttributeView lazyAttributes =
-        Files.getFileAttributeView(path, CloudStorageFileAttributeView.class);
-    assertThat(lazyAttributes.name()).isEqualTo("gcs");
+    try (FileSystem fs = helper.forBucket("red")) {
+      Path path = fs.getPath("/water");
+      Files.write(path, HAPPY, withCacheControl("potato"));
+      CloudStorageFileAttributeView lazyAttributes =
+          Files.getFileAttributeView(path, CloudStorageFileAttributeView.class);
+      assertThat(lazyAttributes.name()).isEqualTo("gcs");
+    }
   }
 
   @Test
-  public void testEquals_equalsTester() {
-    new EqualsTester()
-        .addEqualityGroup(
-            Files.getFileAttributeView(
-                Paths.get(URI.create("gs://red/rum")), CloudStorageFileAttributeView.class),
-            Files.getFileAttributeView(
-                Paths.get(URI.create("gs://red/rum")), CloudStorageFileAttributeView.class))
-        .addEqualityGroup(
-            Files.getFileAttributeView(
-                Paths.get(URI.create("gs://red/lol/dog")), CloudStorageFileAttributeView.class))
-        .testEquals();
+  public void testEquals_equalsTester() throws IOException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2333923612/issues/source?bid=3123107&fileBranchId=3190440#l98)